### PR TITLE
396-errorHandler-not-working-properly

### PR DESCRIPTION
**Pull request by Timmeey**
Fixing issue #396 unexpected ErrorHandling behavior

### CODE REVIEW

File: pippo-core/src/main/java/ro/pippo/core/DefaultErrorHandler.java [hunk 1]
```diff
@@ -171,34 +171,40 @@ protected void renderHtml(int statusCode, RouteContext routeContext) {
...
 
-        ExceptionHandler exceptionHandler = getExceptionHandler(exception);
-        if (exceptionHandler != null) {
-            log.debug("Handling '{}' with '{}'", exception.getClass().getSimpleName(), exceptionHandler.getClass().getName());
-            exceptionHandler.handle(exception, routeContext);
-            return;
-        }
+        if (exception instanceof PippoRuntimeException && exception.getCause() instanceof Exception) {
+            this.handle((Exception) exception.getCause(), routeContext);
+        }else {
```
Comment (decebals) [inline pippo-core/src/main/java/ro/pippo/core/DefaultErrorHandler.java]: space: `} else {`

File: pippo-core/src/test/java/ro/pippo/core/DefaultErrorHandlerTest.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
+package ro.pippo.core;
```
Comment (decebals) [inline pippo-core/src/test/java/ro/pippo/core/DefaultErrorHandlerTest.java]: Add Copyright header (copy/paste from an existed file)

File: pippo-core/src/test/java/ro/pippo/core/DefaultErrorHandlerTest.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.mockito.runners.MockitoJUnitRunner;
+import ro.pippo.core.route.RouteContext;
+
+/**
+ * @author Tim Hinkes (timmeey@timmeey.de)
```
Comment (decebals) [inline pippo-core/src/test/java/ro/pippo/core/DefaultErrorHandlerTest.java]: I think that the name is enough (without email address)

File: README.md [hunk 4]
```diff
@@ -257,3 +257,24 @@ Demo
 ---------------
 Demo applications are available on [pippo-demo](https://github.com/decebals/pippo-demo)   
 For a real life application built with Pippo please look at [Web Accounting - Pippo Demo](https://github.com/decebals/matilda)
+
+Contributors
```
Comment (decebals) [inline README.md]: I cannot maintain a such list. A such list is available at https://github.com/decebals/pippo/graphs/contributors.

**Comment by Timmeey** [comment]
Reviewed/fixed your proposed changes

**Comment by decebals** [comment]
Thanks!

**Pull request merged**